app/services/charts/home/categories_date.rb

```diff
@@ -0,0 +1,27 @@
+class Charts::Home::CategoriesDate < Struct.new(:categories)
+  def date
+    chart_date
+  end
```

Comment:
@marderer no need in creating method just for method. What do you think about renaming `date` to `chart_date`:
```ruby
class Charts::Home::CategoriesDate < Struct.new(:categories)
  def chart_date
     {
       categories_titles: categories_titles,
       categories_amounts: categories_amounts,
       charts_colors: charts_colors
     }
  end
```